test/plugin.test.js

```diff
@@ -287,3 +288,35 @@ QUnit.test('getOptions prioritizes eme options over source options', function(as
     extraOption: 'extra-option'
   }, 'updates source options with eme options');
 });
+
+QUnit.test('removeSession removes sessions', function(assert) {
+  const initData1 = new Uint8Array([1, 2, 3]);
+  const initData2 = new Uint8Array([2, 3, 4]);
+  const initData3 = new Uint8Array([3, 4, 5]);
+  const sessions = [{
+    initData: initData1
+  }, {
+    initData: initData2
+  }, {
+    initData: initData3
+  }];
+
+  removeSession(sessions, initData2);
+  assert.deepEqual(sessions,
```

Comment:
Could throw a few minor negative test cases in here, such as "nothing is removed when no initData is passed", and "nothing is removed when non-matching (matches no items in array) initData is passed".

Reply:
👍